src/views/todo/todo-list.ejs

```diff
@@ -2,19 +2,25 @@
 
 <h1>To Do List</h1>
 
-<form class="to-do-list" action="" method="PUT">
+<form class="to-do-list" action="/" method="PUT">
 
```

Comment:
seems like there's some odd/irregular white space in this file. be consistent in formatting

Reply:
I did "atom beautify" in this case. I will have you show me the white space you are referring to but I think I fixed it :)